vars/getFunctionalTestStage.groovy

```diff
@@ -60,7 +60,7 @@ Map call(Map kwargs = [:]) {
             } else {
                 node(cachedCommitPragma("Test-label${pragma_suffix}", label)) {
                     // Ensure access to any branch provisioning scripts exist
-                    echo "[${name}] Check out from version control"
+                    echo "[${name}] Check out '${base_branch}' from version control"
                     if (base_branch) {
                         checkoutScm(
                             url: 'https://github.com/daos-stack/daos.git',
```

Comment:
Wouldn't it be better to put that inside the if statement with and else clause explaining why?
Or does that not matter?

Reply:
It could have been.  I thought it would ultimately be obvious which branch it took/why based on the value of `$base_branch`.  Wasn't worth pushing a new iteration and waiting for all of the testing on to make that difference at least.